tests/test_demo_scripts.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------------
+# Sense SHORTCUTS
+# ------------------------------------------------------------------------------
+shopt -s expand_aliases
+# Demo scripts test run for each files
+alias demo_calorie_estimation='PYTHONPATH=./ python scripts/run_calorie_estimation.py --weight=65 --age=30 --height=170 --gender=female --path_in=tests/resources/test_video.mp4'
+
+alias demo_fitness_rep_counter='PYTHONPATH=./ python scripts/run_fitness_rep_counter.py --path_in=tests/resources/test_video.mp4'
+
+alias demo_fitness_tracker='PYTHONPATH=./ python scripts/run_fitness_tracker.py --weight=65 --age=30 --height=170 --gender=female --path_in=tests/resources/test_video.mp4'
+
+alias demo_gesture_recognition='PYTHONPATH=./ python scripts/run_gesture_recognition.py --path_in=tests/resources/test_video.mp4'
+
+# ------------------------------------------------------------------------------
+# CHAINED COMMANDS
+# ------------------------------------------------------------------------------
+
+# Run all demo files at once
+test_demos() {
+    echo "Test Calorie Estimation demo ::"
+    demo_calorie_estimation
+
+    echo "Test Fitness Rep Counter demo ::"
+    demo_fitness_rep_counter
+
+    echo "Test Fitness Tracker demo ::"
+    demo_fitness_tracker
+
+    echo "Test Gesture Recognition demo ::"
+    demo_gesture_recognition
+}
+
+# Run the created function
+test_demos
+echo "All tests passed!"
```

Comment:
The script looks a bit overkill to me. How about just this:
```
#!/bin/bash

echo "Test Calorie Estimation demo ::"
PYTHONPATH=./ python scripts/run_calorie_estimation.py --weight=65 --age=30 --height=170 --gender=female --path_in=tests/resources/test_video.mp4

echo "Test Fitness Rep Counter demo ::"
PYTHONPATH=./ python scripts/run_fitness_rep_counter.py --path_in=tests/resources/test_video.mp4

echo "Test Fitness Tracker demo ::"
PYTHONPATH=./ python scripts/run_fitness_tracker.py --weight=65 --age=30 --height=170 --gender=female --path_in=tests/resources/test_video.mp4

echo "Test Gesture Recognition demo ::"
PYTHONPATH=./ python scripts/run_gesture_recognition.py --path_in=tests/resources/test_video.mp4
```

and I wouldn't print "All tests passed!" because I think this will be always be printed, even when if one of the python scripts fails

Reply:
yes, this can be done